lib/canvas/validators/schema_attribute.rb

```diff
@@ -37,6 +37,7 @@ class SchemaAttribute
         "date" => SchemaAttribute::Date,
         "experience_date" => SchemaAttribute::ExperienceDate,
         "experience_slot" => SchemaAttribute::ExperienceSlot,
+        "enquiry_form" => SchemaAttribute::Base,
```

Comment:
[Question] Does this not need registered as a tag like the "base" `form`?

Reply:
No shouldn't do as its not a tag, it'll be an option to the form tag which we don't seem to validate format of in canvas